### PR TITLE
Change dependency version of nekohtml

### PR DESCRIPTION
**Pull request by SachiniSiriwardene**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by nilasini** [comment]
We have to create a new module with new version

**Review by nilasini — CHANGES_REQUESTED** [review]
We have to create a new openid4java module with new version as 1.0.0.wso2v5

**Comment by SachiniSiriwardene** [comment]
> We have to create a new openid4java module with new version as 1.0.0.wso2v5

Added new module as 1.0.0.wso2v6

File: openid4java/1.0.0.wso2v6/pom.xml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                    <artifactId>slf4j-jcl</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>net.sourceforge.nekohtml</groupId>
```
Comment (ThaminduDilshan) [inline openid4java/1.0.0.wso2v6/pom.xml:103]: Shouldn't we use `org.nokogiri`?

File: openid4java/1.0.0.wso2v6/pom.xml [hunk 2]
```diff
@@ -0,0 +1,192 @@
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ThaminduDilshan) [inline openid4java/1.0.0.wso2v6/pom.xml:2]: Let's update to correct license year (2022)

Comment (SachiniSiriwardene) [reply openid4java/1.0.0.wso2v6/pom.xml]: updated

File: openid4java/1.0.0.wso2v6/pom.xml (hunk 1 above)
Comment (SachiniSiriwardene) [reply openid4java/1.0.0.wso2v6/pom.xml]: updated

File: openid4java/1.0.0.wso2v6/pom.xml [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+                        </Embed-Dependency>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <properties>
+        <openid4java.version>1.0.0</openid4java.version>
+        <openid4java.export.version>1.0.0</openid4java.export.version>
+        <net.sourceforge.nekohtml.version>1.9.13</net.sourceforge.nekohtml.version>
```
[the remark this answers is not in the document]
Comment (SachiniSiriwardene) [reply openid4java/1.0.0.wso2v6/pom.xml:174]: updated nekohtml.version

**Review by ThaminduDilshan — APPROVED** [review]

File: openid4java/1.0.0.wso2v6/pom.xml [hunk 4]
```diff
@@ -0,0 +1,192 @@
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (nilasini) [inline openid4java/1.0.0.wso2v6/pom.xml:2]: This is old license. Please use the new one [1]

[1] https://github.com/wso2/orbit/pull/694/files

Comment (SachiniSiriwardene) [reply openid4java/1.0.0.wso2v6/pom.xml]: changed the license

**Review by nilasini — APPROVED** [review]

**Pull request merged**